app/main.py

```diff
@@ -1,8 +1,44 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
```

Comment:
It's ok, but it will be better if you use the same style for parameters according to the init in CarWashStation, it has better readability.